pkcs7/README.md

```diff
@@ -705,6 +706,24 @@ Creating ECC signed bundles:
 ./smime-verify detached-smime-created.p7s ../certs/client-ecc-cert.der content.txt
 ```
 
+
+## PKCS7 Benchmarking
```

Comment:
Was getting error -174 (not compiled in). Tracked it down to `--enable-indef`. Can you make sure and add a note in the docs? Do any other pkcs7 examples need BER encoding too?

```
./benchmark-streaming-envelop
Benchmarking with content size of 10000 bytes
Reading and writing files in chunks of 1000 bytes
Using AES-256 CBC encryption
Using RSA-2048 key

Creating an encoded bundle ... Failed to set stream mode
 : ret = -174
Created file [test-stream-dec.p7b] with size of 0 bytes

Decoding bundle [test-stream-dec.p7b], size of 0 bytes ...  : ret = -174
Processed 0 bytes
```

Reply:
Resolved with this commit (https://github.com/wolfSSL/wolfssl-examples/pull/494/commits/6fe7af7cd09451e1e98a0d8b8a1fed0dd8f5e493). It now prints out a debug message if --enable-indef is not used when building wolfSSL.